releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -36,6 +35,7 @@ export function rcOptionsTypes () {
 export function cliOptionsTypes () {
   return {
     ...rcOptionsTypes(),
```

Comment:
wasn't it coming from here?

Reply:
It seems that the object returned by `rcOptionsType` does not contains `otp` property.

Reply:
Well, I found that there is[ `otp` in the git repository](https://github.com/pnpm/npm-conf/blob/main/lib/types.js#L84), but the actual installation is not(https://www.npmjs.com/package/@pnpm/npm-conf?activeTab=code).

Is there something wrong with the `@pnpm/npm-conf@2.1.1` release ?

Reply:
I have no idea but for sure it should come from there.

Reply:
Could we try to release a new version of `@pnpm/npm-conf`?

 `publish --provenance` also encountered an unknown option error, I added it to `@pnpm/npm-conf` in [ this PR](https://github.com/pnpm/npm-conf/pull/9).